pom.xml

```diff
@@ -113,7 +113,7 @@
       <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <version.spring.boot>2.7.6</version.spring.boot>
+        <version.spring.boot>3.0.1</version.spring.boot>
```

Comment:
For spring-cloud-azure 4.x, the compatible spring-boot version is still 2.7.z.
Refs: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/CHANGELOG.md#450-2022-12-06